scripts/tests/yaml/runner.py

```diff
@@ -202,6 +212,10 @@ def __add_custom_params(self, ctx):
             'server_name': 'chip-app2',
             'server_arguments': '--interactive',
         },
+        'chip-repl': {
+            'adapter': 'matter_yamltest_repl_adapter.adapter',
+            'runner': 'matter_yamltest_repl_adapter.runner',
```

Comment:
Maybe you can add `'use_default_pseudo_clusters': 0` here as well ?

Reply:
This doesn't actually work to override the default though?

Reply:
Discussed over slack. This is something we should fix, but in a follow up PR.

Side note that:
This doesn't actually override the default that sets it to true, when I tested it out. It would work if `use_default_pseudo_clusters`  was a `test_runner_option` and not a `test_parser_option`